resource-managers/kubernetes/core/src/test/scala/org/apache/spark/scheduler/cluster/kubernetes/KubernetesTaskSetManagerSuite.scala

```diff
@@ -21,12 +21,13 @@ import scala.collection.mutable.ArrayBuffer
 import io.fabric8.kubernetes.api.model.{Pod, PodSpec, PodStatus}
 import org.mockito.Mockito._
 
-import org.apache.spark.{SparkContext, SparkFunSuite}
+import org.apache.spark.{SparkConf, SparkContext, SparkFunSuite}
 import org.apache.spark.scheduler.{FakeTask, FakeTaskScheduler, HostTaskLocation, TaskLocation}
 
 class KubernetesTaskSetManagerSuite extends SparkFunSuite {
 
-  val sc = new SparkContext("local", "test")
+  val sc = new SparkContext(master = "local", appName = "test",
+    new SparkConf().set("spark.driver.allowMultipleContexts", "true"))
```

Comment:
why is this change needed?  are we parallelizing tests and running multiple drivers in the same JVM at once?